scribus/scribusapp.cpp

```diff
@@ -213,7 +217,9 @@ void ScribusQApp::parseCommandLine()
 		std::exit(EXIT_SUCCESS);
 	useGUI = true;
 	//We are going to run something other than command line help
-	for(int i = 1; i < argsc; i++) {
+	QString qtARG_SCRIPTARG_PREFIX = QString(ARG_SCRIPTARG_PREFIX);
+	int i = 1;
+	for( ; i < argsc; i++) { //handle all options (not positional parameters)
```

Comment:
i'm not sure that reusing the value of i among the for loops is a good idea.
it's a convention to do `for (int i = 1; i < argsc; i++)` and different behaviors are a pain for people reading the code.
it's effecient for sure... but....